src/oph/ehoks/palaute/tyoelama.clj

```diff
@@ -144,4 +251,34 @@
     (->> (tyopaikkajaksot hoks)
          (filter #(initiate? % hoks opiskeluoikeus))
          (map #(initiate! % hoks suoritus koulutustoimija toimipiste-oid))
+         (filter process?)
+         (map #(process-single-palaute! % hoks))
          doall)))
+
+(defn process-unprocessed!
+  "Creates vastaajatunnus for all herates that are waiting for processing,
+  do not have vastaajatunnus and have heratepvm today or in the past."
+  [_]
+  (log/info "Starting to process unprocessed työelämäpalaute.")
+  (jdbc/with-db-transaction
+    [tx db/spec]
+    (doseq [palaute (get-unprocessed-for-update! tx {})]
+      (process! palaute tx)))
+  (log/info "Done processing unprocessed työelämäpalaute."))
+
+(defn run-scheduler!
+  ^AutoCloseable [start-time rate]
+  (log/info "Starting tep-palaute scheduler.")
+  (let [scheduler
+        (chime/chime-at (chime/periodic-seq start-time rate)
+                        process-unprocessed!
```

Comment:
tälle ilmeisesti ei oo vielä APIa
vakavuus: tiketöitävä

Reply:
Tehty uusi tiksu: https://jira.eduuni.fi/browse/EH-1701